common/djangoapps/student/models.py

```diff
@@ -59,6 +59,26 @@
 AUDIT_LOG = logging.getLogger("audit")
 SessionStore = import_module(settings.SESSION_ENGINE).SessionStore  # pylint: disable=invalid-name
 
+UN_ENROLLED_TO_ALLOWED_TO_ENROLL = 'from unenrolled to allowed to enroll'
+ALLOWED_TO_ENROLL_TO_ENROLL = 'from allowed to enroll to enrolled'
+ENROLL_TO_ENROLL = 'from enrolled to enrolled'
+ENROLL_TO_UN_ENROLL = 'from enrolled to unenrolled'
+UN_ENROLL_TO_ENROLL = 'from unenrolled to enrolled'
+ALLOWED_TO_ENROLL_TO_UN_ENROLL = 'from allowed to enroll to enrolled'
+UN_ENROLL_TO_UN_ENROLL = 'from unenrolled to unenrolled'
+DEFAULT_TRANSITION_STATE = 'N/A'
```

Comment:
If these are strings that get displayed to a user (or administrator), they should be localized.